operators/src/util/raster_stream_to_geotiff.rs

```diff
@@ -226,8 +222,9 @@ mod tests {
         .unwrap();
 
         assert_eq!(
-            include_bytes!("../../../operators/test-data/raster/geotiff_from_stream.tiff")
-                as &[u8],
+            include_bytes!(
+                "../../../operators/test-data/raster/geotiff_from_stream_compressed.tiff"
```

Comment:
do we still need the uncompressed file?

Reply:
currently there is no use for the file. i can remove it if you like.

Reply:
739ca86